### DISCUSSION TITLE
How do I install Gnome?

### DISCUSSION THREAD

**Discussion by mynlive** [minios-linux/minios-live#52] [Q&A] [answered]
I recently switched from **Porteus** (Gnome) to **MiniOS** due to the fact that **Porteus** is on **Slackware**, the question is how to install Gnome?

**Comment by crim50n** [comment, marked as answer]
If you are using 3.x, then `sudo apt2sb -l 3 gnome-core`.
If you are using 4.x, then `sudo apt2sb install -y -l 3 gnome-core`.
You need to put the resulting module on a flash drive and remove the 04-06 modules.
